checker-qual/src/main/java/org/checkerframework/checker/mustcall/qual/NotOwning.java

```diff
@@ -0,0 +1,19 @@
+package org.checkerframework.checker.mustcall.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation indicating that ownership should not be transferred to the annotated parameter, field,
+ * or (when this is written on a method) return type, for the purposes of Must Call checking.
+ *
+ * <p>Parameters and fields are treated as if they have this annotation by default unless they have
```

Comment:
Yes, though it's non-sensical. Consider the must-call types that would result in a caller:
```
@MustCall("close") Object p1;
@MustCall("otherClose") Object p2 = method(p1);
```
`p1` still is the owning pointer to any resource it holds, but calling `method` has created a new (phantom) obligation to also call `otherClose`, which is now owned by `p2`. That is, code like this can only ever create new obligations - it can never destroy them - so soundness is preserved. Writing code like this can (and will) lead to false positives, though; for example, in the code above I don't think there is any way to satisfy `p2`'s obligations.